### DISCUSSION TITLE
Disabling auto pop up of function documentation

### DISCUSSION THREAD

**Discussion by badmagick329** [LazyVim/LazyVim#345] [Q&A] [answered]
Hi all,

 I'm fairly new to lazyvim so apologies if the solution is obvious. I've tried looking for one with no luck. How can I disable the automatic popup of function documentation as I'm writing. What plugin is doing this? (I'm referring to the popup on the right) 
 
![image](https://user-images.githubusercontent.com/63713349/221665129-9722a8e0-e116-433e-bc44-99bf9a4a1c9d.png)

**Comment by folke** [comment, marked as answer]
That's noice.nvim. Set `opts.lsp.signature.auto_open.enabled = false` to disable it

**Reply by badmagick329** [reply]
Thanks. I disabled that but I think it's falling back on another plugin now? The popup is still there but with different syntax highlighting

![image](https://user-images.githubusercontent.com/63713349/221674176-8996d6ea-10db-4005-94ff-5d0b39998759.png)

**Reply by folke** [reply]
Thats completion. Check the nvim-cmp docs on how to disable that

**Reply by badmagick329** [reply]
Awesome thank you!

Found this regarding the issue for anyone else wanting to achieve the same thing
https://github.com/hrsh7th/cmp-nvim-lsp-signature-help/issues/17
